src/brutus-api/brutus_api/tasks.py

```diff
@@ -0,0 +1,41 @@
+from redis import Redis
+from rq import get_current_job
+
+from brutus_api import app
+from brutus_api import nlp
+import json
+
+from watson_developer_cloud import AuthorizationV1
+
+
+classifierName = "brutus-api"
```

Comment:
I think an environment variable would be best. That would make it really easy to change when debugging. Although you can make `brutus-api` the default value in the call to `os.getenv()`.